Makefile.libcompat

```diff
@@ -91,6 +91,11 @@ build${libcompat}: .PHONY
 	${WORLDTMP_MTREE} -f ${.CURDIR}/etc/mtree/BSD.lib${libcompat}.dist \
 	    -p ${WORLDTMP}/usr/lib/debug/usr >/dev/null
 .endif
+# XXX: Remove this hack once morello-llvm supports lib64c
+.if ${libcompat} == "64c" && ${COMPAT_ARCH} == "aarch64"
+	@echo "Morello: creating libcheri link in WORLDTMP"
+	ln -sfnv lib64c ${WORLDTMP}/usr/libcheri
```

Comment:
This seems like it might break incremental Morello builds? Probably needs something in depends-clean.sh or similar, or just bump .require_clean_build?

Reply:
Or perhaps I should just go cherry-pick it to Morello LLVM...

Reply:
I suspect we'll just want to bump .require_clean_build to avoid accidents where people examine the wrong bits in their build trees.

Reply:
I will bump .require_clean_build after merging it (to use the right hash).